File: Deposit_001/07_bits/becxer.cpp
#include <iostream>

using namespace std;

int n, m;
long long arr[100];
int res = 1;

int main(void){
    cin >> n >> m;
    arr[0] = 1;
    for(int i = 1 ; i <= m ; i++){
        arr[i] = (n-i+1) * arr[i-1] / i;
        res += arr[i];
    }   
    
    cout << res << endl;
}

File: ProblemPool/sortgame/becxer.py
n = int(raw_input())
arr = map(int, str(raw_input()).split())
k = int(raw_input())

max_val = 0
next_max = 0
max_idx = 0
for i in range(k):
    kab = map(int, str(raw_input()).split())
    a = kab[0]
    b = kab[1]
    

res = ''
for i in range(n):
    res += str(arr[i]) + ' '
print res
